drivers/dma/dma_mcux_lpc.c

```diff
@@ -44,8 +44,11 @@ struct dma_mcux_lpc_dma_data {
 	struct call_back *data_cb;
 	uint32_t *channel_index;
 	uint32_t num_channels_used;
+	char is_inputmux_setup;
```

Comment:
Sure, I have made this change but as your aware and for future supporters, when we go to implement multiple chaining for the dma channel chaining, this driver will need a variable to keep track of the current OTrigs currently in use.

Reply:
yes and it will also need to keep track of what is connected to what to be able to disconnect things appropriately, so you would need more than this anyways

Reply:
This is being resolved in this PR now